spec.html

```diff
@@ -24705,13 +24712,13 @@ <h1>ECMAScript Standard Built-in Objects</h1>
+  <p>Each built-in function defined in this specification is created by calling the CreateBuiltinFunction abstract operation (<emu-xref href="#sec-createbuiltinfunction"></emu-xref>). The values of the _length_ and _name_ parameters are the initial values of the *"length"* and *"name"* properties as discussed below.</p>
   <p>Every built-in function object, including constructors, has a *"length"* property whose value is a non-negative integral Number. Unless otherwise specified, this value is equal to the number of required parameters shown in the subclause headings for the function description. Optional parameters and rest parameters are not included in the parameter count.</p>
   <emu-note>
     <p>For example, the function object that is the initial value of the *"map"* property of the Array prototype object is described under the subclause heading &laquo;Array.prototype.map (callbackFn [ , thisArg])&raquo; which shows the two named arguments callbackFn and thisArg, the latter being optional; therefore the value of the *"length"* property of that function object is 1.</p>
   </emu-note>
   <p>Unless otherwise specified, the *"length"* property of a built-in function object has the attributes { [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *true* }.</p>
-  <p>Every built-in function object, including constructors, has a *"name"* property whose value is a String. Unless otherwise specified, this value is the name that is given to the function in this specification. Functions that are identified as anonymous functions use the empty String as the value of the *"name"* property. For functions that are specified as properties of objects, the name value is the property name string used to access the function. Functions that are specified as get or set accessor functions of built-in properties have *"get "* or *"set "* prepended to the property name string. The value of the *"name"* property is explicitly specified for each built-in functions whose property key is a Symbol value. The *"name"* property is set using SetFunctionName.</p>
+  <p>Every built-in function object, including constructors, has a *"name"* property whose value is a String. Unless otherwise specified, this value is the name that is given to the function in this specification. Functions that are identified as anonymous functions use the empty String as the value of the *"name"* property. For functions that are specified as properties of objects, the name value is the property name string used to access the function. Functions that are specified as get or set accessor functions of built-in properties have *"get "* or *"set "* prepended to the property name string. The value of the *"name"* property is explicitly specified for each built-in functions whose property key is a Symbol value.</p>
```

Comment:
All engines agree that the `name` property for built-in accessors have `"get "` or `"set "` prepended, depending on the type of accessor.

For Symbol-named getters like `@@toStringTag`, their `name` properties are indeed explicitly defined.

Reply:
So the idea here is that rather than using the `prefix` argument to `SetFunctionName`, the `name` parameter for such functions should be understood to include the `get` part already. (Since Symbol-named builtins have their name fully specified, they don't need to hit the `Type(_name_) is Symbol` path in `SetFunctionName`.)

Works for me.

Reply:
The issue there is that engines are allowed to omit the `prefix` from the <code><var>func</var>.[[InitialName]]</code> internal slot.

Reply:
Whether they omit or include it, i think this suggestion is still valid.

Reply:
I am inclined to say that that's not worth worrying about.